README.md

```diff
@@ -49,7 +49,7 @@ Build everything:
 In the root directory:
 
 ```
-npx lerna bootstrap
+yarn install
```

Comment:
@nick-verida Is it wort adding `nvm use` here?

Reply:
It is in the section Developer Notes -> Node Version